conference.js

```diff
@@ -2024,6 +2034,77 @@ export default {
                             eventData
                         });
                     }
+
+                    if (eventData.name === CAPTURE_SCREENSHOT_MESSAGE) {
+                        APP.store.dispatch(captureLargeVideoScreenshot()).then(dataURL => {
+                            if (!dataURL) {
+                                room.sendEndpointMessage(sender._id, {
+                                    name: SEND_SCREENSHOT_MESSAGE,
+                                    id: undefined
+                                });
+
+                                return;
+                            }
+
+                            const splitImage = {
+                                uuId: uuidv4(),
+                                chunks: []
+                            };
+                            let currentIndex = 0;
+
+                            while (currentIndex < dataURL.length) {
+                                const newIndex = currentIndex + IMAGE_CHUNK_SIZE;
+
+                                splitImage.chunks.push(dataURL.slice(currentIndex, newIndex));
+                                currentIndex = newIndex;
+                            }
+
+                            const size = splitImage.chunks.length;
+
+                            for (const [ idx, chunk ] of splitImage.chunks.entries()) {
```

Comment:
I think we could simplify this by chunking and sending in a single loop. Can you give it a try?

Reply:
done